packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -40,6 +41,7 @@ contract LongShortPair is Testable, Lockable {
         uint64 expirationTimestamp; // Unix timestamp of when the contract will expire.
         uint256 collateralPerPair; // How many units of collateral are required to mint one pair of synthetic tokens.
         bytes32 priceIdentifier; // Price identifier, registered in the DVM for the long short pair.
+        bool enableEarlyExpiration; // Enables the LPS contract to be settled early.
```

Comment:
Can you replace all "LPS" in this contract with "LSP"? Sorry to nit pick...

Reply:
yes! that's not a nit pick sir, that's a great call out!